i3pystatus/openstack_vms.py

```diff
@@ -0,0 +1,69 @@
+from i3pystatus import IntervalModule
+from .core.util import round_dict
+# requires python-novaclient
+from novaclient.v2 import client
+import webbrowser
+
+
+class Openstack_vms(IntervalModule):
+    """
+    Displays the number of VMs in an openstack cluster in ACTIVE and
+    non-ACTIVE states.
+    """
+
+    settings = (
+        ("auth_url", "OpenStack cluster authentication URL (OS_AUTH_URL)"),
+        ("username", "Username for OpenStack authentication (OS_USERNAME)"),
+        ("password", "Password for Openstack authentication (OS_PASSWORD)"),
+        ("tenant_name", "Tenant/Project name to view (OS_TENANT_NAME)"),
+        ("color", "Display color when non-active VMs are =< `threshold` "
+        "(default: #00FF00"),
```

Comment:
You don't need to manually document the defaults, they are automatically documented.